tests/rustdoc-ui/check-cfg.stderr

```diff
@@ -6,5 +6,3 @@ LL | #[cfg(uniz)]
    |
    = note: `#[warn(unexpected_cfgs)]` on by default
 
-warning: 1 warning emitted
```

Comment:
This looks wrong. We should still be emitting the number of warning. I think this is because you called `reset_err_count`.